df_auth/drf/serializers.py

```diff
@@ -126,27 +128,70 @@ class FirstLastNameSerializerMixin(serializers.Serializer):
 
 
 class SocialTokenObtainSerializer(FirstLastNameSerializerMixin, TokenCreateSerializer):
```

Comment:
OAuth1 and OAuth2 have different input params `access_token` and  `oauth_token + oauth_token_secret `. So, it's better to have 2 different serializers (and views) for them. Then `access_token`, `oauth_token` and `oauth_token_secret` will be required fields. 

So, lets keep `/social` URL for OAuth2 and add `/social/oauth1` for OAuth1